p4-16/psa/psa.p4

```diff
@@ -236,10 +234,8 @@ extern Meter<S> {
   @ControlPlaneAPI
   {
     reset(in MeterColor_t color);
-    setParams(in S committedRate, in S committedBurstSize
-              in S peakRate, in S peakBurstSize);
-    getParams(out S committedRate, out S committedBurstSize
-              out S peakRate, out S peakBurstSize);
+    setParams(in MeterConfig);
+    getParams(out MeterConfig);
```

Comment:
The DirectMeter has the ability to set parameters independently for each entry, via the 'in TableEntry entry' parameter to setConfig.

I would have guessed that this same level of configurability would be possible for the Meter extern, by having say an 'index' parameter, maybe type uint32 if some other type is not more appropriate.